<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevSetting_RT1052_Poe_AP60_Bluetooth</class>
 <widget class="QWidget" name="DevSetting_RT1052_Poe_AP60_Bluetooth">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMP Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     border-radius: 4px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }
QPushButton:checked{
	background-color: rgb(0, 178, 0);
}</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>330</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-Out1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-In1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>N-In1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Control</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="NIn1_Channel_Gain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Channel_Mute">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NIn1_Channel_Eq">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eq</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NIn1_Channel_Mute">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Channel_Eq">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eq</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AIn1_Channel_Gain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Control</string>
   </property>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="AIn1_Delay">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Phase">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Phase</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_xover">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>xover</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AIn1_Gain">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Eq">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eq</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Dyn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>DYN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Mute">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_delay_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>239.021ms</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="VolumeWidget" name="Meter_A_1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>N1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_N_1" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>71</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_AL" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Bluetooth Control</string>
   </property>
   <widget class="QPushButton" name="bt_play">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_yinliangjian">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_yinliangjia">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_duankai">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_chonglian">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重连</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_peidui">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>配对</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_shangyishou">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_xiayishou">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QLabel" name="bt_yinliang">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>54</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bt_zhuangtai">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>可连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>customWidget/volumewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
